encoding/thrift/thriftrw-plugin-yarpc/fake_testreporter_test.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"fmt"
+	"runtime"
+	"runtime/debug"
+
+	"github.com/golang/mock/gomock"
+)
+
+// FakeTestStatus states how a fake TestingT finished.
+type FakeTestStatus int
```

Comment:
Didn't we just implement this somewhere else.. no action for this diff, but should we make a separate repo for a fake testing.T?